### ISSUE TITLE
[BUG]: Lower quality than the examples on the demo page

### ISSUE THREAD

**Issue by GalenMarek14** [justinjohn0306/MaskGCT-Windows#8]
## Describe the bug
After following the installation instructions and using the same examples from the demo page, I was unable to replicate the quality of the examples. For example, the whispering voice always outputs something between a whisper and a normal voice. I tried both the inference script and the Gradio app, with the same result. Additionally, the duration calculator seems to be broken for Chinese - it makes the output twice as fast when set to auto.

This is the demo page result:
https://vocaroo.com/15JxVNPRScwD

This is mine:
https://vocaroo.com/1d62HKhiGTsB

## How To Reproduce
Follow the instructions to install on Win 11 with special phonemizer and generate audio

## Expected behavior
Quality should be the same as the examples

## Screenshots
![image](https://github.com/user-attachments/assets/82656b55-4c5d-4565-93ad-b58979243656)

## Environment Information
 - Operating System: Windows 11
 - Python Version: Python 3.10.15
 - Driver & CUDA Version: Driver 546.92 & CUDA 12.4
 - Error Messages and Logs: Posted it above, here quoted version:

> ./models/tts/maskgct/g2p\sources\g2p_chinese_model\poly_bert_model.onnx
> D:\AIMaskGCTTTS\WINDOWS_VERSION\MaskGCT-Windows\app.py:97: FutureWarning: You are using `torch.load` with `weights_only=False` (the current default value), which uses the default pickle module implicitly. It is possible to construct malicious pickle data which will execute arbitrary code during unpickling (See https://github.com/pytorch/pytorch/blob/main/SECURITY.md#untrusted-models for more details). In a future release, the default value for `weights_only` will be flipped to `True`. This limits the functions that could be executed during unpickling. Arbitrary objects will no longer be allowed to be loaded via this mode unless they are explicitly allowlisted by the user via `torch.serialization.add_safe_globals`. We recommend you start setting `weights_only=True` for any use case where you don't have full control of the loaded file. Please open an issue on GitHub for any issues related to this experimental feature.
>   stat_mean_var = torch.load("./models/tts/maskgct/ckpt/wav2vec2bert_stats.pt")
> D:\AIMaskGCTTTS\WINDOWS_VERSION\venv\lib\site-packages\torch\nn\utils\weight_norm.py:143: FutureWarning: `torch.nn.utils.weight_norm` is deprecated in favor of `torch.nn.utils.parametrizations.weight_norm`.
>   WeightNorm.apply(module, name, dim)
> * Running on local URL:  http://127.0.0.1:7860
>
> To create a public link, set `share=True` in `launch()`.
> D:\AIMaskGCTTTS\WINDOWS_VERSION\venv\lib\site-packages\whisper\__init__.py:150: FutureWarning: You are using `torch.load` with `weights_only=False` (the current default value), which uses the default pickle module implicitly. It is possible to construct malicious pickle data which will execute arbitrary code during unpickling (See https://github.com/pytorch/pytorch/blob/main/SECURITY.md#untrusted-models for more details). In a future release, the default value for `weights_only` will be flipped to `True`. This limits the functions that could be executed during unpickling. Arbitrary objects will no longer be allowed to be loaded via this mode unless they are explicitly allowlisted by the user via `torch.serialization.add_safe_globals`. We recommend you start setting `weights_only=True` for any use case where you don't have full control of the loaded file. Please open an issue on GitHub for any issues related to this experimental feature.
>   checkpoint = torch.load(fp, map_location=device)
> Building prefix dict from the default dictionary ...
> Loading model from cache C:\Users\Ozkan\AppData\Local\Temp\jieba.cache
> Loading model cost 0.651 seconds.
> Prefix dict has been built successfully.
> D:\AIMaskGCTTTS\WINDOWS_VERSION\MaskGCT-Windows\models\tts\maskgct\g2p\g2p\chinese_model_g2p.py:100: UserWarning: Creating a tensor from a list of numpy.ndarrays is extremely slow. Please consider converting the list to a single numpy.ndarray with numpy.array() before converting to a tensor. (Triggered internally at C:\actions-runner\_work\pytorch\pytorch\builder\windows\pytorch\torch\csrc\utils\tensor_new.cpp:281.)
>   batch_label_starts = torch.tensor(batch_label_starts, dtype=torch.long)

## Additional context
Thank you very much for this project
